### ISSUE TITLE
Deploy prod - pre-launch just dev-ops thing

### ISSUE THREAD

**Issue by psineur** [TheRealGD/therealgd#12]
This is basically that we need to figure out:
 * how we store configuration (not in public repo lol)
 * what boxes and where
 * docker or no docker (I vote no docker, since it's PITA to develop code inside container)
* ~~Configure optional auto-deploy to web server as part of Jenkins build~~

**Comment by psineur** [comment]
This isn't really blocked by #10 and kinda runs in parallel with #9 - we just need to make sure shit is compatible. So no AWS-only tools or services, as manual and close to bare metal as possible

**Comment by psineur** [comment]
Coupled with #10 and #6 (although 6 can be completed as just set of instructions / scripts to make backup work, with DB url configurable from ENV variables/etc)

**Comment by jstretch85** [comment]
> how we store configuration (not in public repo lol)

GitHub's individual developer plan is only $7/mo. Would that work? Could just take the repo private.

**Comment by psineur** [comment]
@jstretch85 totally would work

**Comment by psineur** [comment]
Add “copy legal pages from GitHub wiki to PM wiki once prod dB is established”.

**Comment by DEVxALMIGHTY** [comment]
Added "Configure optional auto-deploy to web server as part of Jenkins build". If this doesn't belong here, lmk.

**Comment by DEVxALMIGHTY** [comment]
Making separate issue for task "Configure optional auto-deploy to web server as part of Jenkins build" 

See #57

**Comment by psineur** [comment]
- [ ] how we store configuration (not in public repo lol)
- [ ] what boxes and where
- [ ] docker or no docker (I vote no docker, since it's PITA to develop code inside container)
- [ ] Configure optional auto-deploy to web server as part of Jenkins build

**Comment by psineur** [comment]
Jenkins needs to update for docker - #163 
Boom! :boom:
